governance.md

```diff
@@ -0,0 +1,149 @@
+# Governance
+
+Coraza is an open-source WAF solution compatible with seclang and CoreRuleset.
+
+The Coraza project aims for open and transparent governance and decision-making, thus encouraging community building and contribution.
+
+A formal governance structure helps us to
+
+- Provide a structure for individuals to become involved in the project.
+- Communicate all processes for members to operate within the project.
+- Document a system for open product development, roadmapping, and planning.
+- Provide a means for making decisions if consensus cannot be reached.
+
+This document describes the rules and governance of the Coraza project. It is meant to be followed by all the developers of the project and the Coraza community.
+
+## Community overview
+
+On a high level, the key moving parts of the community are:
+
+- **GitHub activity** (issues + pull requests)
+- **Slack community**
+  - [`#coraza`](https://owasp.slack.com/archives/C02BXH135AT) channel is where all the discussions and questions happen in the coraza community
+  - [`#coraza-core`](https://owasp.slack.com/archives/C034AF28KPV) channel is where internal discussions around the team happen.
+- **Monthly meetings** (last wednesday of every month) to give an update on the open topics in coraza and held votings for new team members. It's being held in the `#coraza` channel.
+
+## Values
+
+Coraza community strives for trust, respect, kindness, giving feedback effectively, and building a welcoming environment. The Coraza developers generally decide by consensus.
+
+## Projects
+
+Each project must have a `CODEOWNERS` file with at least one code owner. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the `#coraza` channel. Any new projects should be first proposed on the `#coraza-core` channel following the voting procedures listed below.
+
+## Decision making
+
+Coraza is a meritocratic, consensus-based community project.
+
+### Team members
+
+Team member status may be given to those who have made ongoing contributions to the Coraza project for at least 2 months. This is usually in the form of significant code improvements and/or notable work on documentation.
+
+New members may be proposed by any existing member in the monthly meeting. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [majority vote](#majority-vote).
+
+If the new member proposal is accepted, the proposed team member will be pinged in a github issue on this repository and confirm its acceptance.
+
+Team members may retire at any time by opening a github issue in this repository requesting for removal.
+
+Team members can be removed by [majority vote](#majority-vote) opening a github issue and voting in the next 48 hours.
```

Comment:
Since we do have Slack as a backchannel, I'd suggest removal not be in a public issue as it can be sensitive

Reply:
Removing people is a painful process for any project. It needs to be there, but I don't have a clear idea of how to do it. Do we know how other open-source projects do this?

Reply:
It should consider a on-boarding and off-boarding process